packages/api-client/src/api/cmsBlocks/index.ts

```diff
@@ -4,11 +4,20 @@ import { CmsBlockQuery, CmsBlockQueryVariables } from '../../types/GraphQL';
 import cmsBlocks from './cmsBlocks';
 import { Context } from '../../types/context';
 
-export default async (
+/**
+ * Fetch CMS Blocks from Magento Api.
+ *
+ * @param context - VSF Context
+ * @param identifiers - identifiers of CMS blocks
+ * @param customQuery - (optional) - custom GraphQL query that extends the default cmsBlocks query
```

Comment:
```suggestion
 * @param [customQuery] - custom GraphQL query that extends the default cmsBlocks query
```
https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

Reply:
thanks for the link, but keep in mind that we don't use jsdoc. We use https://api-extractor.com/pages/setup/invoking/

Reply:
The optionality information is in TypeScript anyways so perhaps it makes sense to just textually describe the param, without any extra info

Reply:
I checked and it doesn't work so I can't change that.

Reply:
@Frodigo can you just change it to without "(optional)"? If the return type can be inferred then the fact that the argument is optional should be too

Reply:
TS can infer if an argument is optional but the doc generator or the reading user - can't. I want to know if something is optional BEFORE I event starts to write the code.

Reply:
No, I mean since it could infer the return type in a way that Marcin felt is acceptable, I'm wondering if the same can happen for the argument

Reply:
Return type is easy to see without docs annotation, but the optional argument isn't clear without that so I recommend to keep that information.